src/string.cr

```diff
@@ -3927,7 +3960,7 @@ class String
   end
 
   private def dump_char(char, error, io)
-    dump_or_inspect_char(char, error, io) do
+    dump_or_inspect_char char, error, io do
```

Comment:
I really dislike people making pointless changes which inflate diffs.

It's just as bad as people making pointless comments on PRs about style.

Reply:
I wouldn't make that randomly. Just six lines above in an almost identical method defintion was literally the exact same line with different formatting. That felt too odd, really.